snapper/Snapper.cc

```diff
@@ -99,6 +114,8 @@ namespace snapper
 
 	filesystem = Filesystem::create(*config_info, root_prefix);
 
+	syncSelinuxContexts();
```

Comment:
I wonder if always trying to sync the contexts slows down startup. Did you do some tests for that with many snapshots?